packages/evolution-backend/src/api/survey.participant.routes.ts

```diff
@@ -83,7 +82,13 @@ export default (authorizationMiddleware, loggingMiddleware: InterviewLoggingMidd
                         ['id', 'uuid', 'is_valid', 'is_completed', 'response', 'participant_id']
                     );
                 }
-                res.status(200).json({ status: 'success', interview });
+
+                // Check if interview is frozen, if so, do not allow access
+                if (interview?.is_frozen) {
+                    console.log('activeSurvey: Interview is frozen');
+                    return res.status(403).json({ status: 'failed', error: 'interview is frozen' });
```

Comment:
Ici, status pourrait être forbidden et le message "Interview cannot be accessed", c'est plus générique. is_frozen est un détail d'implémentation qui risque de peut-être disparaître étant donné qu'on a is_active et is_frozen qui existent et veulent dire la même chose.